utils/hoursUtils.js

```diff
@@ -126,7 +125,9 @@ function parseTodayHours(todayOpenHours, todayCloseHours) {
 }
 
 function getTodayWeekday() {
-    return DateTime.now().toUTC().setZone(AMERICA_CHICAGO).weekday
+    const weekday = DateTime.now().toUTC().setZone(AMERICA_CHICAGO).weekday
```

Comment:
This is not the most elegant solution, but is a quick fix

Reply:
Fair enough.  Although it makes more sense then the elegant-but-cryptic `weekday % 0`